extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -61,9 +62,13 @@ export class AmpAdUIHandler {
   /**
    * Create a default placeholder if not provided.
    * Should be called in baseElement createPlaceholderCallback.
-   * @return {*} TODO(#23582): Specify return type
+   * @return {?Element}
    */
   createPlaceholder() {
+    if (isNewLoaderExperimentEnabled(this.element_)) {
```

Comment:
What's the current plan on the new loader experiment? Curious what data we collect from the experiment. 
cc @keithwrightbos Do you see any potential impact on ads side with the default loader change? Thanks.

Reply:
The current plan is to have a ramp up schedule of 1%, 10%, 50%, and then 100% with a minimum one week spacing at each stage to verify metrics are not impacted beyond what we might reasonably expect and to understand the scope of the impact.

@nainar Who might have more context.

Reply:
What's our decision on the loader and placeholder. If we decide to keep the ad icon as the placeholder, should we keep the old logic here because we will always need the ad icon

Reply:
The new loader renders the "ad" icon as a part of the loader (in the middle of the spinner). So in that case, we don't want to create the placeholder here. Eventually, we should remove the default placeholder logic.

Reply:
I see. Thanks
In this case, can I assume the ad icon will still be displayed on top of the publisher provided placeholder.

Reply:
Yep, we'll display a circle with a dark (but translucent) grey background on top of the placeholder, then the spinner / logo on top of that.

Reply:
Got it. Thanks